libs/domain/content/src/services/default-font.service.ts

```diff
@@ -7,13 +7,23 @@ export class DefaultFontService implements FontService {
 
   install(text: string): void {
     this.collectFonts(text).forEach((font) => {
-      this.pageMetaService.add({
-        name: 'link',
-        attrs: {
-          rel: 'stylesheet',
-          href: `https://fonts.googleapis.com/css2?family=${font}:wght@400;500;600;700&display=swap`,
+      this.pageMetaService.add([
+        {
+          name: 'link',
+          attrs: {
+            rel: 'preload',
+            href: `https://fonts.googleapis.com/css2?family=${font}:wght@400;500;600;700&display=swap`,
+          },
         },
-      });
+        {
+          name: 'link',
+          attrs: {
+            rel: 'stylesheet',
+            href: `https://fonts.googleapis.com/css2?family=${font}:wght@400;500;600;700&display=swap`,
+            media: 'all',
+          },
+        },
+      ]);
```

Comment:
Is there a reasonable design thinkable to do the heavy lifting of adding preload (and potentially other things) lower level? I'd prefer to have an api that is doing this stuff for you, as this is quite specific knowledge.